README.md

```diff
@@ -21,38 +21,23 @@ Or install it yourself as:
 ## Usage
 
 ```ruby
-require 'gctime'
->> GCTime.total_time
-=> 0.0
->> GCTime.enable
-=> nil
+>> GC.measure_total_time
+=> true
 >> 4.times { GC.start }
 => 4
->> GCTime.total_time
-=> 127.42100000000012 # ms
->> GCTime.disable
-=> nil
+>> GC.stat(:time)
+=> 123 # milliseconds (returns a Float on JRuby and TruffleRuby)
```

Comment:
Actually it's an Integer for all Rubies, isn't it?
I checked JRuby and TruffleRuby, it's an Integer for both:
```
ruby -ve 'GC.start; p GC.stat :time'
truffleruby 22.0.0-dev-f439b388, like ruby 3.0.2, GraalVM CE Native [x86_64-linux]
5
```

Reply:
Oh wow, not sure why I was certain of this. Thanks, I'll update the readme.

Reply:
uh https://github.com/Shopify/gctime/runs/4266747810?check_suite_focus=true#step:5:45

```
GCTimeTest#test_total_time_is_an_integer [/home/runner/work/gctime/gctime/test/gctime_test.rb:14]:
Expected 0.0 to be an instance of Integer, not Float.
```

Reply:
Argh, I'm an idiot, nevermind.